src/resolutions/lesson-02/7-object-keys-typing.ts

```diff
@@ -0,0 +1,19 @@
+import { Equal, Expect } from "../../../helper";
+import { expect, it } from "vitest";
+
+// 7. Object.keys correct typing
+
+const typedObjectKeys = <T extends string>(obj: Record<T, number>) => {
+  return Object.keys(obj) as T[];
+};
+
```

Comment:
solo una pequeña nota, en esta función en particular solo no importan las keys, podemos  hacer 
``obj: Record<T, any> solo a manera de dar entender que `:any` no es malo del todo.

Reply:
Thx, addressed in next commit.